main.tf

```diff
@@ -44,6 +44,7 @@ module "ocp_base" {
   addons                          = var.addons
   manage_all_addons               = var.manage_all_addons
   verify_worker_network_readiness = var.verify_worker_network_readiness
+  cluster_config_endpoint_type    = var.cluster_config_endpoint_type
```

Comment:
TFLint in ./:
1 issue(s) found:

Warning: [Fixable] variable "cluster_config_endpoint_type" is declared but not used (terraform_unused_declarations)

  on variables.tf line 330:
 330: variable "cluster_config_endpoint_type" {

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.5.0/docs/rules/terraform_unused_declarations.md